package.json

```diff
@@ -1,48 +1,36 @@
-        "d2-utils-cypress": "./bin/d2-utils-cypress"
-    },
-    "main": "src/index.js",
-    "version": "2.2.0",
-    "author": "Viktor Varland <viktor@dhis2.org>",
-    "license": "BSD-3-Clause",
-    "private": false,
-    "engines": {
-        "node": ">=10"
-    },
-    "scripts": {
-        "lint": "yarn lint:js && yarn lint:text",
-        "lint:js": "d2-style js check",
-        "lint:text": "d2-style text check",
-        "lint:staged": "yarn lint:js --staged && yarn lint:text --staged",
-        "format": "yarn format:js && yarn format:text",
-        "format:js": "d2-style js apply",
-        "format:text": "d2-style text apply",
-        "format:staged": "yarn format:js --staged && yarn format:text --staged",
-        "build": "d2-utils-docsite build ./docs -o ./dist --jsdoc src/ --jsdocOutputFile api.md",
-        "start": "d2-utils-docsite serve ./docs -o ./dist --jsdoc src/ --jsdocOutputFile api.md"
-    },
-    "dependencies": {
-        "@dhis2/cli-helpers-engine": "^1.5.0",
-        "concurrently": "^5.2.0",
-        "cross-spawn": "^7.0.1",
-        "cypress": "^4.0.2",
-        "cypress-cucumber-preprocessor": "^2.0.1",
-        "fs-extra": "^8.1.0",
-        "inquirer": "^7.0.4",
-        "wait-on": "^4.0.2"
-    },
-    "publishConfig": {
-        "access": "public"
-    },
-    "devDependencies": {
-        "@dhis2/cli-style": "^6.0.0",
-        "@dhis2/cli-utils-docsite": "^1.3.0"
+  "private": true,
+  "workspaces": ["./packages/*"],
+  "author": "Viktor Varland <viktor@dhis2.org>",
+  "devDependencies": {
+    "@dhis2/cli-style": "^7.0.0"
+  },
+  "contributors": [
+    {
+      "name": "Jan-Gerke Salomon",
+      "email": "jgs.salomon@gmail.com",
+      "url": "https://functionalprogramming.ninja"
     }
+  ],
+  "scripts": {
+    "commands:build": "cd ./packages/commands && yarn build",
+    "commands:lint": "cd ./packages/commands && yarn lint",
+    "commands:lint:js": "cd ./packages/commands && yarn lint:js",
+    "commands:lint:text": "cd ./packages/commands && yarn lint:text",
+    "commands:lint:staged": "cd ./packages/commands && yarn lint:staged",
+    "commands:format": "cd ./packages/commands && yarn format",
+    "commands:format:js": "cd ./packages/commands && yarn format:js",
+    "commands:format:text": "cd ./packages/commands && yarn format:text",
+    "commands:format:staged": "cd ./packages/commands && yarn format:staged",
+    "cli:lint": "cd ./packages/cli && yarn lint",
+    "cli:lint:js": "cd ./packages/cli && yarn lint:js",
+    "cli:lint:text": "cd ./packages/cli && yarn lint:text",
+    "cli:lint:staged": "cd ./packages/cli && yarn lint:staged",
+    "cli:format": "cd ./packages/cli && yarn format",
+    "cli:format:js": "cd ./packages/cli && yarn format:js",
+    "cli:format:text": "cd ./packages/cli && yarn format:text",
+    "cli:format:staged": "cd ./packages/cli && yarn format:staged",
+    "cli:build": "cd ./packages/cli && yarn build",
+    "cli:start": "cd ./packages/cli && yarn start"
```

Comment:
While these are technically fine, they're quite repetitive and need to be synced with the scripts in each workspace package.  Yarn allows you to run a command in an individual workspace like this `yarn workspace @dhis2/cli-utils-cypress format:stages` (unfortunately it matches on the package name, not the directory) or on all workspaces with `yarn workspaces format:staged`.  Maybe it's enough to just have these scripts at the root?

```js
"build": "yarn workspaces build",
"start": "yarn workspace @dhis2/cypress-helpers build && yarn workspace @dhis2/cli-utils-cypress start",
"format": "yarn workspaces format",
"lint": "yarn workspaces lint"
```

Or something similar?

Reply:
I've moved them all up to the root